src/Aspire.Dashboard/wwwroot/css/app.css

```diff
@@ -150,3 +150,7 @@ fluent-data-grid-cell .long-inner-content {
 .validation-message {
     color: var(--error);
 }
+
+.page-header {
+    padding: calc(var(--design-unit) * 1px) calc(var(--design-unit) * 2px);
```

Comment:
how does 2 calc work together?

Reply:
Its two values on padding, e.g. `padding: 1px 2px`. One value is for top/bottom, the other is for left/right.

Reply:
`padding` is a shorthand property for `padding-top`, `padding-right`, `padding-bottom` and `padding-left`. When only two values are given, the first is `padding-top`/`padding-bottom` and the second is `padding-left`/`padding-right`. So in this case I didn't change the top/bottom padding, they were already set to DesignUnit, I just doubled the left/right padding to DesignUnit * 2.